tests/ui/borrowck/borrowck-assign-to-andmut-in-aliasable-loc.stderr

```diff
@@ -17,8 +17,8 @@ LL |     *s.pointer += 1;
    |
 help: consider changing this to be a mutable reference
    |
-LL | fn c(s: &mut &mut S<'_>) {
-   |         ~~~~~~~~~~~~~~~
+LL | fn c(s: &mut  &mut S) {
+   |          +++
```

Comment:
There are a couple suggestions where there is an extra space inserted. While this looks wrong, it is correct because the extra space exists in the source code as well.

Reply:
Yeah, that's ok. We already rely on people running rustfmt themselves.